packages/docs/community/release-process/smart-contracts.md

```diff
@@ -72,11 +72,23 @@ yarn verify-deployed -n $NETWORK -b $PREVIOUS_RELEASE -f
 
 ### Check Backward Compatibility
 
-Use the following script to compile candidate and current release contracts to check storage layout and ABI backwards compatibility, subject to the following exceptions:
+This script performs some automatic checks to ensure that smart contract
+versions have been updated correctly.
+
+Specifically, it compiles candidate and current release contracts to check storage layout and ABI backwards compatibility, subject to the following exceptions:
```

Comment:
Formatting of this list seems to be off, see:
https://docs.celo.org/community/release-process/smart-contracts